README.md

```diff
@@ -140,7 +140,7 @@ See the [Usage Guide](#usage-guide) for more information on how to get started u
     - [features.rememberMe](#featuresrememberme)
     - [features.autoFocus](#featuresautofocus)
     - [features.disableAutocomplete](#featuresdisableautocomplete)
-  - [cspNonce](#cspNonce)
+  - [cspNonce](#cspnonce)
```

Comment:
Was this intentional? It seems like the header is camelCase https://github.com/okta/okta-signin-widget/pull/3469/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L1365